scania-digital-design-system/sdds
# AB#804 Feature/header

**Describe pull-request**  
- Add header template from KY Header

**How to test**
- Check in storybook both mobile and desktop view
- To show side menu, switch toggle on Storybook control.

**Missing component (to be added as next feature)**
- User account styling (circle profile, and different positioning on mobile)

**Solving issue**  

Fixes: [AB#804](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/804)
